docs/polaris/applications-and-libraries/libraries/math-libraries.md

```diff
@@ -6,9 +6,10 @@ Some math libraries targeting CPUs are made available as part of the `nvhpc` mod
 
 - BLAS & LAPACK can be found in the `$NVIDIA_PATH/compilers/lib` directory.
 - ScaLAPACK can be found in the `$NVIDIA_PATH/comm_libs` directory.
-
+- GNU Scientific Library, [GSL-2.7](https://www.gnu.org/software/gsl/) available as `module help gsl`
+- AMD Optiming CPU Libraries, [AOCL v4.0](https://www.amd.com/en/developer/aocl.html) available as `module help aocl`
+- Other Cray-based math libs such as Libsci, FFTW were made available by `module load cray-libsci/cray-fftw`
```

Comment:
Think users would attempt to actually run `module load cray-libsci/cray-fftw` and be confused when it complains about unknown module?

Reply:
Addressed in the below changes